alerter.js

```diff
@@ -1,26 +1,49 @@
+const { expect } = require("chai");
+const environment = "PRODUCTION";
+
 let alertFailureCount = 0;
+let failureCount = 0;
+/*Setting threshold to 100 deg celcuis*/
+/*if the celcuis value is less than 100 no alert is sent. */
+/*if the celcuis value is greater than 100 alert is sent. */
 
 function networkAlertStub(celcius) {

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.